.ci/run-tests

```diff
@@ -10,7 +10,7 @@ fi
 set -euxo pipefail
 
 
-TEST_SUITE=${TEST_SUITE-oss}
+TEST_SUITE=${TEST_SUITE-xpack}
```

Comment:
Switched to `xpack` by default because the test suite doesn't pass with `oss` due to ML.